presto-testng-services/src/main/java/io/prestosql/testng/services/ReportUnannotatedMethods.java

```diff
@@ -129,4 +130,18 @@ public static boolean isTemptoClass(Class<?> aClass)
     {
         return aClass.getPackage().getName().startsWith("io.prestosql.tempto");
     }
+
+    private static void forceTestngFailure(Runnable runnable)
```

Comment:
Please see https://github.com/prestosql/presto/pull/4951 for my take on this.

Reply:
I guess #4951 is more robust as it covers more listeners.